File: app/src/main/java/Model/Message.java
package Model;

import java.util.Objects;

public class Message {
    String from, to,  message;
    long seconds;

    public Message() {
    }

    public String getFrom() {
        return from;
    }

    public void setFrom(String from) {
        this.from = from;
    }

    public String getTo() {
        return to;
    }

    public void setTo(String to) {
        this.to = to;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public long getSeconds() {
        return seconds;
    }

    public void setSeconds(long seconds) {
        this.seconds = seconds;
    }

    @Override
    public int hashCode() {
        return Objects.hash(getFrom(), getTo(), getMessage(), getSeconds());
    }
}

File: app/src/main/java/com/whiteside/insta/model/Message.kt
package com.whiteside.insta.model

import java.util.*

class Message {
    var from: String? = null
    var to: String? = null
    var message: String? = null
    var seconds: Long = 0
    override fun hashCode(): Int {
        return Objects.hash(from, to, message, seconds)
    }
}
